sherpa/utils/err.py

```diff
@@ -279,7 +279,7 @@ class IOErr(SherpaErr):
             'badarray': "'%s' must be a Numpy array, list, or tuple",
             'notascii': "file '%s' does not appear to be ASCII",
             'noparamcols': 'No parameter columns found in %s',
-            'toomanycols': "Received more column names: %s, than required: %s",
+            'wrongcolnumber': "Expected %d columns but found %d",
```

Comment:
@hamogu: a very trivial point but the connotation I get from the mnemonic 'wrongcolnumber' sounds like an indexing error ( 'ie. indexed column 0 instead if column 1'). Perhaps something like 'wrongnumcols' would improve clarity.

Reply:
Done. (Just pushed.)